### PR TITLE
update email for small invoices

### PR DESCRIPTION
**Pull request by nickpell**
http://manage.dimagi.com/default.asp?163246

@benrudolph @orangejenny

### CODE REVIEW

**Comment by benrudolph** [comment]
this looks good to me. happy to merge whenever label is off and tests are done

**Comment by orangejenny** [comment]
looks fine to me

File: corehq/apps/accounting/models.py [hunk 1]
```diff
@@ -1714,11 +1715,20 @@ def is_email_throttled(self):
 
     def email_context(self):
         context = super(BillingRecord, self).email_context()
+        total_balance = sum(invoice.balance for invoice in Invoice.objects.filter(
+            is_hidden=False,
+            subscription__subscriber__domain=self.invoice.get_domain(),
```
Comment (orangejenny) [inline corehq/apps/accounting/models.py]: is the double underscore some kind of convention?

Comment (nickpell) [inline corehq/apps/accounting/models.py]: it's django query syntax: https://docs.djangoproject.com/en/1.8/topics/db/queries/#field-lookups

Comment (orangejenny) [inline corehq/apps/accounting/models.py]: thanks!

**Comment by nickpell** [comment]
QA feedback needs to be implemented before merge

**Comment by nickpell** [comment]
@benrudolph Good to go

**Pull request merged**